src/Map/index.js

```diff
@@ -70,6 +74,15 @@ export default class Map extends Component {
     }
   }
 
+  async componentDidUpdate(prevPops) {
+    const { markerCollection, editor } = prevPops
```

Comment:
Why is `prevProps` used in this method instead of the current props in `this.props`?

Reply:
Is that not what I'm supposed to do? The documentation makes it seem like I'm supposed to the `prevProps` that this hook receives.

Reply:
Do you mean [react's documentation](https://reactjs.org/docs/react-component.html#componentdidupdate) or something else? If react, those docs show the API. Since `componentDidUpdate` get's invoked whenever props or state change, both the previous props/state and the current props/state are made available. The current props in `this.props` and the current state in `this.state`. My question is just confirming that we intend to pass the stale props to `this.loadAddresses` instead of the values from after the update. Is that correct here?

Reply:
Current props should be ok.